akvo/rsr/dir/app/utils/misc.js

```diff
@@ -99,3 +100,5 @@ export const getLogo = logo => logo
 export const getFirstPhoto = photos => (photos && photos.length)
   ? photos.slice(0, 1).pop()
   : null
+
+export const isPartnerSites = () => (window.location.href.indexOf('localakvoapp') !== -1 || window.location.href.indexOf('akvoapp') !== -1)
```

Comment:
You can use `location.hostname` for this: https://developer.mozilla.org/en-US/docs/Web/API/Location/hostname

e.g 

```js
window.location.hostname.endsWith("akvoapp.com") || window.location.hostname === "localakvoapp"
```

And it should be `isPartnerSite`